src/displayapp/screens/Paddle.cpp

```diff
@@ -50,6 +52,11 @@ void Paddle::Refresh() {
   // checks if it has touched the side (right side)
   if (ballX >= LV_HOR_RES - ballSize - 1) {
     dx *= -1;
+    dy += rand() % 3 - 1; // add a little randomization in wall bounce direction, one of [-1, 0, 1]
+    if (dy > 5) // limit dy to be in range [-5 to 5]
```

Comment:
Doesn't follow the brace style used elsewhere.

Reply:
fixed with https://github.com/InfiniTimeOrg/InfiniTime/pull/797/commits/14fb7abb9af1c3e24b6d64f3c80b14596ea06c24